### DISCUSSION TITLE
Engine location ignored, only thruster location is used.

### DISCUSSION THREAD

**Discussion by huyandenghuangbo** [JSBSim-Team/jsbsim#754] [Q&A]
I launched a simulation which run the rascal.xml, but it shows: Engine location ignored, only thruster location is used. Like tihs 

![微信图片_20221031141145](https://user-images.githubusercontent.com/55626173/198943001-3e5e6013-d2ea-4cbc-9325-b444c62119a1.png)
and my plane can't take off, is this the reason that it can't takeoff?

**Comment by seanmcleod** [comment]
The warning that the engine location and orientation are ignored and the thruster's location and orientation are used won't be the cause of any take-off issues. Ensure that the thruster location and orientation make sense.

**Comment by bomber64** [comment]
Show us lines 104 and onwards
